project/plugins.sbt

```diff
@@ -1,11 +1,5 @@
-libraryDependencies ++= Seq(
-    "com.google.collections" % "google-collections" % "0.8",
-    "org.codehaus.plexus"    % "plexus-utils"       % "1.5.4",
-    "org.slf4j"              % "slf4j-api"          % "1.6.1",
-    "org.slf4j"              % "slf4j-simple"       % "1.6.1")
+resolvers += Resolver.sbtPluginRepo("releases")
 
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.13.0")
```

Comment:
The assembly stuff is still quite fragile, and unfortunately we don't have tests for it yet... you'll have to start up Hadoop and everything just to see that it works. Does this version bump break anything, or is it fine?
We should probably have a Hadoop docker image just to test that this job works...